lib/sensors/src/MoistureSensor.cpp

```diff
@@ -15,6 +15,11 @@ constexpr int Sensor_Status_Reset      = 0x7F;
 
```

Comment:
I'm currently updating the sensor classes to use an abstract class. As such I don't think you'll need to worry about editing this file as I will eventually overwrite it.

Reply:
Sounds good, Younes had me looking into writing code for a new driver for the moisture sensor. Does this mean you will be taking that on instead of me?

Reply:
Hm If its a new driver that doesn't sound like it will be me. The current firmware was written by Christopher. @younesr1 can you confirm if a new sensor will need code or if this code needs to be redone?

Reply:
nah no new driver is being written now so felix ur right, ur code will just overwrite this

Reply:
in that case ill leave the file as is for the time being